ThunderCloud/BundleDiagnosticsTableViewController.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  BundleDiagnosticsTableViewController.swift
+//  ThunderCloud
+//
+//  Created by Simon Mitchell on 08/04/2020.
+//  Copyright © 2020 threesidedcube. All rights reserved.
+//
+
+import UIKit
+import ThunderTable
+
+class BundleDiagnosticTableViewController: TableViewController {
+    
+    let dateFormatter = DateFormatter()
```

Comment:
I was going to leave a comment on this because `viewDidLoad` is still pretty clean! 
But thinking about it, sometimes things do happen in `UIViewControllers` before `viewDidLoad` is called (might not be the case here!). So `lazy` instantiation will protect against this not being fully initialised until `viewDidLoad`.